lib/tasks/sap_update_authorizations.rake

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+namespace :sap do
+  desc "Updates users metadata in authorizations by SAP"
+  task update_metadata: :environment do
+    users = Decidim::User.all
+    users.find_each do |user|
+      authorization = Decidim::AuthorizationHandler.handler_for(
+        "sap_authorization_handler",
+        user: user
+      )
+      Decidim::CreateSapAuthorization.call(authorization) do
+        Rails.logger.info "UpdateMetadata: -------------------"
```

Comment:
I think this and the closing comment should go before line 12, not sure how the content of this block is executed...